src/python/serenity_types/pricing/derivatives/options/valuation.py

```diff
@@ -277,9 +278,3 @@ class OptionValuationResult(CamelModel):
     Partial derivative of the spot notional value of the contract with respect to theta X 1 day,
     expressed in base currency.
     """
-
-    warnings: Optional[List[str]]
-    """
-    Serenity generated warnings, e.g. the specified OptionValuation.option_asset_id is invalid,
-    and so it is excluded from option valuation.
-    """
```

Comment:
Why are warnings being removed in this PR?

Reply:
Ah yes, I realized that this warnings should be in risk.services response object.
So that we can gather all warnings and consolidate there.